packages/admin-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@builder.io/admin-sdk",
-  "version": "0.0.8",
+  "version": "0.0.9-0",
```

Comment:
Why the -0?

Reply:
These are dev releases for the customer to test/verify. Actual merge will have the final non `-n` version.